portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,75 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private List<String> quotes;
+
+   public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Here is one message!");
+    quotes.add("Here is a second message!");
+    quotes.add("Here is a third message!");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String json = convertToJson(quotes);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+    //response.getWriter().println("<h1>Hello Emily!</h1>");
+  }
+
+    /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+   //have to use a for loop, convert ArrayList to json adding keys
+  private String convertToJson(List<String> listt) {
+    String json = "{";
```

Comment:
So usually in comments you don't have to explain what you're trying to do technically (eg. using a for loop), but just on a higher level. Like here, saying "Convert ArrayList to a Json string with numerical keys" is sufficient :)

You'd also want to move this comment inside the method instead of leaving it out here.